data/org.freedesktop.impl.portal.Clipboard.xml

```diff
@@ -0,0 +1,181 @@
+    <!-- 
+        RequestClipboard:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+
+        Requests clipboard access for the given portal session. This request
+        must be made before the session starts.
+
+        This portal does NOT create it's own session. Instead, it offers existing sessions
+        created from other portals the option to integrate with the clipboard. For whether
+        this interface is supported for a given session, refer to that portal's documentation.
+        See #org.freedesktop.portal.RemoteDesktop to integrate clipboard with the 
+        remote desktop session.
+    -->
+    <method name="RequestClipboard">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+    </method>
+    <!-- 
+        SetSelection:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+
+        Sets the owner of the clipboard formats in 'mime_types' in @options to
+        the session, i.e. this session has data for the advertised clipboard formats.
+
+        See #org.freedesktop.portal.FileTransfer to transfer files using the
+        'application/vnd.portal.filetransfer' mimetype.
+
+        May only be called if clipboard access was given after starting the session.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>mime_types as</term>
+            <listitem><para>
+              A list of mime types that the session has clipboard content for.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+     -->
+    <method name="SetSelection">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+    </method>
+    <!--
+        SelectionWrite:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @serials: The list of serials of requests that this answer is directed towards
+        @request_fds: A dict of serials to UnixFD handles, where the key of each entry is the
+                      serial of a request and the value is the corresponding handle of the
+                      contents of the selection being written to
+
+        Answer to 'SelectionTransfer' signal(s). Transfers the clipboard content for one or more 
+        serials via file descriptors.
+        It is the Callee that creates the file descriptors.
+
+        May only be called if clipboard access was given after starting the session.
+    -->
+    <method name="SelectionWrite">
+      <annotation name="org.gtk.GDBus.C.UnixFD" value="true"/>
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a(u)" name="serials" direction="in"/>
```

Comment:
This is an array of structs (tuples) where each one has only one element, like this:

```
[(1234,), (4567,), ...]
```

Are you sure that's what you want? Structs with only one element are usually not very useful, for the same reason that a Python `Tuple[int]` is not very useful: you could just use the int directly.

If you just want an array of integers:

```
[1234, 4567, ...]
```

then the type for that is `au`.